web-animations-1/Overview.bs

```diff
@@ -3290,7 +3296,7 @@ To produce <a>computed keyframe offsets</a>, we define a procedure to
 
 <a>Computed keyframes</a> are produced using the following procedure.
 Note that this procedure is only performed on a <a>keyframe effect</a> having
-a <a>target element</a> for which computed property values can be calculated.
+a existing <a>effect target</a> for which computed property values can be calculated.
```

Comment:
I think if you're going to introduce "existing" here, I think you need to link to a definition of what it means for a target to exist.

Reply:
Existing means present in the layout tree, such as a `::before` with a content property, as opposed to one without content that doesn't appear. The idea is to hang Animations off of their target Elements which have a well-defined lifecycle and make Animation lifecycle independent of the ill-defined pseudo-element lifecycle. The Animation always runs and has a visible effect iff the pseudo-element does.

Reply:
That still sounds like existing wants some sort of definition.

Reply:
Maybe just remove "existing" since "for which computed property values can be calculated" does the job. As this procedure is affine, it can always be skipped when there is nothing to style.

Reply:
Ok, makes sense.